.drone.yml

```diff
@@ -138,11 +138,6 @@ steps:
 
 - name: nctl-upgrade-test
   <<: *buildenv
-  environment:
```

Comment:
Is this accidental or on purpose? I am out of the loop here :)

Looks like some branching shenanigans going on?

Reply:
I did a `git merge` with upstream/dev because we needed the change which undeprecated the `std` feature gate on casper-types. I don't know exactly why it's showing up like this and not a single merge commit.